images-optimize-and-upload-cf7/frontend/assets/script.js

```diff
@@ -100,6 +101,31 @@ jQuery(document).ready(function ($) {
         errorHandler() //hide error
         countImages = 0
       }, false);
+
+      // callback invalid input
+      document.addEventListener('wpcf7invalid', function (event) {
+        const uploadErr = arrayObjHasPropertyValue(event.detail.apiResponse.invalid_fields, 'field', NAME_TAG);
+
+        if (uploadErr) {
+          errorHandler(uploadErr);
```

Comment:
Your function arrayObjHasPropertyValue may return FALSE, and that value goes into the errorHandler(false).

Reply:
You mean if the upload field has the message set as "FALSE"?